### DISCUSSION TITLE
[Feature] <Replaceable maps>

### DISCUSSION THREAD

**Discussion by malanxi2022** [immich-app/immich#6043] [Feature Request]
### The feature

The current maps are so ugly and require a vpn to access in China. Can you replace it with Amap, I like the map style of icloud albums. And it's accurate.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
You can configure a custom style.json URL in the admin settings.

**Reply by arrio464** [reply]
Sorry to bother you with a stupid question. 
Are there any examples or docs to help me in customizing it?
I read [get-map-style](https://immich.app/docs/api/get-map-style), but I don't quite understand it.
I would be grateful if you could give me a helping hand.
